client/src/components/AddBoardDialog/AddBoardDialog.tsx

```diff
@@ -55,6 +55,7 @@ const AddBoardDialog = ({ onAddNewBoard }: Props): JSX.Element => {
     if (!error) {
       setOpenAlert({ open: true, message: 'Board created', severity: 'success' });
       onAddNewBoard(board);
+      handleClose();
       return;
     }
```

Comment:
I assume `onAddNewBoard` is an async function; would it be better to close _after_ successful add to ensure showing the dialog occurs only on a successful request (not sure how `onAddNewBoard` works though so you might've already taken care of that!)

Reply:
This function is not async, because it just adds a board that was given by a previous async call to the board array. You have a good eye for possible problems, though!

Reply:
Sweet. That makes sense!

Reply:
Looks good to me